scripts/generatePagesFromConsolidatedJSON.py

```diff
@@ -817,6 +829,15 @@ def write_index_step2(findex, cpt, cptHasCode, cptVariants):
    cpt=0;
    cptHasCode=0;
    cptVariants=0
+   step2data = dict()
+
+   step2data["2014"]    = [0,0,0,0,0,0,0,0]
```

Comment:
we should start using a list of years instead of explicit values in the scripts

Reply:
Issue https://github.com/GraphicsReplicability/replicability.graphics/issues/30 added.